packages/core/src/lib/init.ts

```diff
@@ -65,7 +65,7 @@ export async function init({
   options: InternalOptions
   cookies: cookie.Cookie[]
 }> {
-  const { providers, provider } = parseProviders({
+  const { providers, provider } = await parseProviders({
```

Comment:
This would mean we are hitting the `well-known` endpoints on every auth requests now, compare to the previous approach where we only hit it on getting the authorization URL.

Reply:
we could cache it in the module scope